doc/getting-started.md

```diff
@@ -0,0 +1,175 @@
+# Getting Started
+
+This section will serve as a simple Mio tutorial. It will start from
```

Comment:
There's a school of thought that suggests not using words like 'simple' or 'easy' in teaching material: if your reader gets stuck, they feel extra extra bad, and are more likely to drop off, because it's _not_ simple to them: that's why they're reading a tutorial!

(this is of course very very very minor)